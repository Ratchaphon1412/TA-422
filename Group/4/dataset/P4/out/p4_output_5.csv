Team,Engagements
teamRed,755
teamGreen,1422
teamBlue,776
teamYellow,676
teamMagenta,746
teamCyan,557
