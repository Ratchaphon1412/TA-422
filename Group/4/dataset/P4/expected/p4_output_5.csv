Hashtag,Likes,Retweets,TotalEngagement
teamGreen,1111,311,1422
teamBlue,542,234,776
teamRed,505,250,755
teamMagenta,461,285,746
teamYellow,428,248,676
teamCyan,393,164,557
